palettable/tableau/tableau.py

```diff
@@ -17,6 +17,7 @@
 palette_type = 'qualitative'
 
 palette_names = [
+    'Tableau_10_New'
```

Comment:
This needs a comma to end the line, but otherwise I think this looks good!
```suggestion
    'Tableau_10_New',
```